### DISCUSSION TITLE
Installing YoloV5 using pip

### DISCUSSION THREAD

**Discussion by ajrenzo** [ultralytics/ultralytics#2157] [Q&A]
This is probably a dumb question, but considering the large amount of fake pypi uploads containing malware lately I figured I'd ask just to be safe. Is the "pip install yolov5" legit from Ultralytics or is that something someone else uploaded? it seems rather new and I didnt find anything on the Ultralytics github page about it.

**Comment by glenn-jocher** [comment]
@ajrenzo hello there! 👋

Great question, and it's always wise to be cautious about what you install. The `pip install yolov5` command is indeed associated with Ultralytics and is a legitimate way to install the YOLOv5 package. However, since your question is about YOLOv5, I'd like to gently remind you that this repo is focused on YOLOv8.

For YOLOv8, you can refer to our official documentation at https://docs.ultralytics.com for guidance on installation and usage. Always ensure you're following the official documentation to avoid any unofficial or potentially malicious software.

Stay safe and happy coding! 😊
